<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screencap</class>
 <widget class="QMainWindow" name="Screencap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Part Video Recording Station</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnConfig">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStation">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>                      STATION</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnStart">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnStop">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDMCNo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>  DMC NUMBER</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPow">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>   Powered By Mangnetivity Solutions</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnoption1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnoption2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblDMCNo_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>325</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>abc</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
